core/services/keystore/keys/dkgsignkey/key.go

```diff
@@ -0,0 +1,92 @@
+package dkgsignkey
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math/big"
+
+	"go.dedis.ch/kyber/v3"
+	"go.dedis.ch/kyber/v3/group/edwards25519"
+)
+
+var suite = edwards25519.NewBlakeSHA256Ed25519()
+
+// Raw represents a raw dkgsign secret key in little-endian byte order.
+type Raw []byte
+
+// Key returns a Key object from this raw data.
+func (r Raw) Key() Key {
+	privKey := suite.Scalar().SetBytes(r)
+	key, err := keyFromScalar(privKey)
+	if err != nil {
+		panic(err) // should never happen
+	}
+	return key
+}
+
+func (r Raw) String() string {
+	return "<DKGSign Raw Private Key>"
+}
+
+func (r Raw) GoString() string {
+	return r.String()
+}
+
+// Key is DKG signing key that conforms to the keystore.Key interface
+type Key struct {
+	privateKey     kyber.Scalar
+	publicKeyBytes []byte
+	PublicKey      kyber.Point
+}
+
+// New creates a new DKGSign key
+func New() (Key, error) {
+	privateKey := suite.Scalar().Pick(suite.RandomStream())
```

Comment:
Yup, no need to worry about endianness here, the library will pick an appropriate private key given the source of randomness. The endianness comes into play only when we marshal to bytes and unmarshal from bytes, since if we marshal in big-endian and unmarshal in little-endian the expression `unmarshal(marshal(key)) == key` will evaluate to false.